diff --git a/node_modules/react-native-map-clustering/lib/ClusteredMapView.js b/node_modules/react-native-map-clustering/lib/ClusteredMapView.js
index 8c7820c..2de764b 100644
--- a/node_modules/react-native-map-clustering/lib/ClusteredMapView.js
+++ b/node_modules/react-native-map-clustering/lib/ClusteredMapView.js
@@ -136,9 +136,9 @@ const ClusteredMapView = forwardRef(
         const bBox = calculateBBox(region);
         const zoom = returnMapZoom(region, bBox, minZoom);
         const markers = superCluster.getClusters(bBox, zoom);
-        if (animationEnabled && Platform.OS === "ios") {
-          LayoutAnimation.configureNext(layoutAnimationConf);
-        }
+        // if (animationEnabled && Platform.OS === "ios") {
+        //   LayoutAnimation.configureNext(layoutAnimationConf);
+        // }
         if (zoom >= 18 && markers.length > 0 && clusterChildren) {
           if (spiralEnabled) updateSpiderfier(true);
         } else {
